src/plugins/auto/src/plugin.cpp

```diff
@@ -566,7 +566,7 @@ std::list<DeviceInformation> MultiDeviceInferencePlugin::GetValidDevice(
         }
     }
 
-    // Priority of selecting device: dGPU > VPUX > iGPU > MYRIAD > CPU
+    // Priority of selecting device: dGPU > NPU > iGPU > MYRIAD > CPU
     std::list<DeviceInformation> devices;
     if (networkPrecision == "INT8") {
         devices.splice(devices.end(), VPUX);
```

Comment:
Please update device name from `VPUX` to `NPU`.

Reply:
Done.